src/AutoRest.CSharp/Mgmt/AutoRest/MgmtOutputLibrary.cs

```diff
@@ -23,7 +23,7 @@ internal class MgmtOutputLibrary : OutputLibrary
         private Dictionary<OperationGroup, ResourceOperation>? _resourceOperations;
         private Dictionary<OperationGroup, ResourceContainer>? _resourceContainers;
         private Dictionary<string, ResourceData>? _resourceData;
-        private Dictionary<string, Resource>? _armResource;
+        private Dictionary<OperationGroup, Resource>? _armResource;
```

Comment:
put back as string and take OperationGroup into the Resource constructor

Reply:
Done